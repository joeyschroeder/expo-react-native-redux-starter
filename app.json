{
  "expo": {
    "name": "Blank Template",
    "slug": "my-new-project",
    "privacy": "public",
    "platforms": ["ios", "android", "web"],
    "version": "1.0.0",
    "orientation": "portrait",
    "icon": "./src/images/icon.png",
    "splash": {
      "image": "./src/images/splash.png",
      "resizeMode": "contain",
      "backgroundColor": "#ffffff"
    },
    "updates": {
      "fallbackToCacheTimeout": 0
    },
    "assetBundlePatterns": ["**/*"],
    "ios": {
      "supportsTablet": true
    }
  }
}
