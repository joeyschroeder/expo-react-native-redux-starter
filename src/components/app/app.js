import { StyleSheet, Text, View } from 'react-native';

import React from 'react';

const styles = StyleSheet.create({
  container: {
    alignItems: 'center',
    flex: 1,
    justifyContent: 'center'
  }
});

export const App = () => {
  return (
    <View style={styles.container}>
      <Text>Open up app.js to start working on your app!</Text>
    </View>
  );
};
