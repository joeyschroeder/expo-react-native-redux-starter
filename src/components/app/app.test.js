import { render, screen } from '@testing-library/react-native';
import { App } from './app';

describe('<App />', () => {
  it('should render correctly', () => {
    render(<App />);

    const component = screen.toJSON();
    expect(component).toMatchSnapshot();
  });
});
