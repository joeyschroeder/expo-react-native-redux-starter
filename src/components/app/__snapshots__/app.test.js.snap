// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Open up app.js to start working on your app!
  </Text>
</View>
`;
